src/pages/sta/cartesian_coordinates.astro

```diff
@@ -7,15 +7,297 @@ import BlueText from "../../components/BlueText.astro"
+<SubSection title="Notation">
+
+    <p>
+        Some textbooks differentiate between <em>free
+        vectors</em>, which are free to slide around, and
+        <em>bound vectors</em>, which are anchored in space. We
+        will only use free vectors.
+      </p>
+
+      <p>
+        We will use the over-arrow notation <InlineEquation equation="\\vec{a}" /> for vector
+        quantities. Other common notations include bold
+        <InlineEquation equation="\\boldsymbol{a}" /> and under-bars <InlineEquation equation="\\underline{a}" />. For unit
+        (length one) vectors we will use an over-hat <InlineEquation equation="\\hat{a}" />.
+      </p>
+</SubSection>
+
+<SubSection title="Cartesian coordinate system">
+
+    <p>
+        Cartesian coordinates (also known as rectangular
+        coordinates) are the regular \(x,y\) or \(x,y,z\) coordinates in
+        2D and 3D, with perpendicular straight-line axes.
+      </p>
+      <Center>
+      <table id="rvp-tc" class="mb-3">
+        <tr class="border-below">
+          <th>coordinate</th>
+          <th>name</th>
+          <th>range</th>
+          <th>definition</th>
+        </tr>
+        <tr>
+          <td>\(x\)</td>
+          <td>horizontal</td>
+          <td>\(-\infty \lt x \lt \infty\)</td>
+          <td>horizontal distance from the \(y\)-axis</td>
+        </tr>
+        <tr>
+          <td>\(y\)</td>
+          <td>vertical</td>
+          <td>\(-\infty \lt y \lt \infty\)</td>
+          <td>vertical distance from the \(x\)-axis</td>
+        </tr>
+      </table>
+    </Center>
+      <p>
+        In 3D the \(x\),\(y\),\(z\) axes are oriented in <em>right-handed
+        order</em>, so that fingers curling from \(x\) to \(y\) means
+        the thumb points in the \(z\) direction.
+      </p>
+    <Center>
+        <PrairieDrawCanvas id="rvp-fc-c" width="450" height="450">
+            <p class="figureCaption">
+                Cartesian coordinate system, showing the origin \(O\) and
+                three positions with their coordinates.
+            </p>
+        </PrairieDrawCanvas>
+    </Center>
+
+    <CalloutContainer slot="cards">
+        <CalloutCard title="Notation note">
+            <p>
+                When writing coordinates we can either list the coordinate
+                values explicitly, like \(x = 2\), \(y = 1\), or we can just
+                write the numbers in an ordered list, like \( (2,
+                1) \).
+              </p>
+              <p>
+                Writing them as a list assumes that we know which
+                coordinate system we are using, and which order the
+                coordinates are in. We will typically write the coordinate
+                values explicitly to avoid confusion. This also applies to
+                <a href="rvv.html#rvv-ic">vector components</a>.
+              </p>
+        </CalloutCard>
+
+        <CalloutCard>
```

Comment:
I know that this is copied from Dynamics, which is generally fine. However, I think we probably don't want to copy the "Did you know?" callout cards. These are more "fun facts" and I think it's going to be a bit weird to have them show up in multiple courses?